webpack-test/ConfigTestCases.template.js

```diff
@@ -88,10 +90,17 @@ const describeCases = config => {
 						const cacheDirectory = path.join(outBaseDir, ".cache", testSubPath);
 						let options, optionsArr, testConfig;
 						beforeAll(() => {
-							options = prepareOptions(
-								require(path.join(testDirectory, "webpack.config.js")),
-								{ testPath: outputDirectory }
-							);
+							expect(() => {
+								try {
+								options = prepareOptions(
+									require(path.join(testDirectory, "webpack.config.js")),
```

Comment:
The test can crash in some webpack configs, e.g. 

https://github.com/web-infra-dev/rspack/blob/d3ddbe7d2c3bca2995998a3365c375e19ce8eb7f/webpack-test/configCases/plugins/profiling-plugin/webpack.config.js#L7